README.md

```diff
@@ -395,6 +395,17 @@ This book inspired me at the age of 14 to get a tattoo I ended up getting 7 year
 
 > Princess Irene's great-grandmother has a testing task for Curdie. He will not go alone though, as she provides him with a companion -- the oddest and ugliest creature Curdie has ever seen, but one who turns out to be the most loyal friend he could have hoped for.
 
+#### [The Wool Trilogy](https://www.goodreads.com/book/show/20745447-the-wool-trilogy) (2013) *by [Hugh Howey](https://en.wikipedia.org/wiki/Hugh_Howey) [4.34]
+
+This serie has been an amazing companion for some time, with vibrating caracters and a great rythm. The organization of the silo is fascinating and the story keeps so many surprises I never grew bored. This is everything you need in a trilogy to face a quarantine time!
```

Comment:
Thanks! Could you add your name to this, like the other reviews, and also spell check it?